src/main/java/org/cancogenvirusseq/muse/api/ApiDefinition.java

```diff
@@ -198,4 +198,23 @@
       consumes = MediaType.APPLICATION_JSON_VALUE,
       method = RequestMethod.GET)
   ResponseEntity<Flux<DataBuffer>> download(@Valid @RequestBody DownloadRequest downloadRequest);
+
+    @ApiOperation(
+            value = "Download molecular data as a single .fasta.gz gzip compressed file",
```

Comment:
are we including the song analysis metadata with the download?

Reply:
Do you mean the tsv records?? no...